extensions/functions_string.yaml

```diff
@@ -50,7 +50,9 @@ scalar_functions:
     name: substring
     description: >-
       Extract a substring of a specified `length` starting from position `start`.
-      A `start` value of 1 refers to the first characters of the string.
+      A `start` value of 1 refers to the first characters of the string.  When
+      `length` is not specified the function will extract a substrait starting
```

Comment:
```suggestion
      `length` is not specified the function will extract a substring starting
```
that is my favorite typo this week.

Reply:
That was not the plan I had in mind.  At least it wasn't substrate